GoogleTranslate/googletranslate.ini

```diff
@@ -27,6 +27,12 @@
 # * Default: en
 #output_lang = en
 
+# Time that the plugin will wait before sending the request.
+# * Time in seconds (can be used with float type)
+# * The minimum value is 0.25
+# * Default: 0.25
+#waiting_time = 0.25

```

Comment:
Same remarks than for WebSuggest:
* hard-coded maximum value of 3 seconds
* I think `idle_time` is more appropriate than `waiting_time`